python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -569,7 +570,9 @@ def _get_code_pointer_dict_from_kwargs(kwargs: ClickArgMapping) -> Mapping[str,
         return {
             cast(
                 RepositoryDefinition,
-                repository_def_from_target_def(loadable_target.target_definition),
+                repository_def_from_target_def(
+                    loadable_target.target_definition, DefinitionsLoadType.ROOT
```

Comment:
INITIAL?

Reply:
DEFINING?

Reply:
Thinking about the language here.

"When booting a Dagster process that contains definitions, one of two things are happening.

1. You are defining a set of definitions for Dagster to ingest, effectively updating Dagster's understanding of the world at a point in time.
2. You are reconstructing Dagster's definitions that were previously defined

This distinction is important when the source of truth for a Dagster definition is state (for example in a hosted ingestion or BI tool accessable via an API). The state should only be accessed when defining definitions, whereas when reconstructing definitions that state should be retrieved from Dagster's metastore."